app/assets/javascripts/manage_submissions.js

```diff
@@ -44,19 +82,25 @@ $(document).ready(function () {
 
     // Fetch data and render it in the modal 
     get_score_details(course_user_datum_id).then((data) => {
-
       const problem_headers = data.submissions[0].problems.map((problem) => {
         const max_score = problem.max_score;
-        const autograded = problem.grader_id < 0 ? " (Autograded)" : "";
+        const autograded = problem.grader_id == null || problem.grader_id < 0 ? " (Autograded)" : "";
         return `<th class="submission-th">
                   ${problem.name}
                   <br>
                   <i> ${max_score} ${autograded} </i>
                 </th>`;
       }).join('');
 
+      tweaks = [];
+
       const submissions_body = data.submissions.map((submission) => {
-        
+        const Tweak = new AutolabComponent(`tweak-value-${submission.id}`, { amount: null });
+        Tweak.template = function () {
+          return Edit( this.state.amount );
+        }
+        tweaks.push({tweak: Tweak, submission_id: submission.id});
```

Comment:
Can we just include the entire submission here? i.e. `tweaks.push({tweak: Tweak, submission: submission});` Then ideally we can get rid of `get_submission_details`

Reply:
can include entire submission here, but we would still need some function to get the updated tweak total so that it will update without refreshing